tf-modules/cumulus_ecs_service/README.md

```diff
@@ -1,4 +1,35 @@
-Note: To prevent a race condition during service deletion, make sure to set
+# Cumulus - ECS service module
+
+**Note:** To prevent a race condition during service deletion, make sure to set
 depends_on to the related aws_iam_role_policy; otherwise, the policy may be
 destroyed too soon and the ECS service will then get stuck in the DRAINING
 state.
+
+## Included resources
+
+Provides an ECS service and task definition, including autoscaling configuration and Cloudwatch alarms for monitoring.
+
+## Input variables
+
+See [variables.tf](./variables.tf) for the input variables to this module and the default values for optional variables.
+
+## Outputs
+
+- **service_name** - Name of the created ECS service
+
+## Example
+
+```hcl
```

Comment:
I'm on the fence about including these. I think it is nice to have a simple example of usage right here in the README, especially because our deployment documentation doesn't show how to integrate this module. But I guess our example deployment does show it being used?